### PR TITLE
STFU: Apply and watch regorules

### PR DESCRIPTION
**Pull request by teozkr**

### CODE REVIEW

File: rust/operator-binary/Cargo.toml [hunk 1]
```diff
@@ -11,6 +11,7 @@ version = "0.3.0-nightly"
 [dependencies]
 stackable-operator = { git = "https://github.com/stackabletech/operator-rs.git", tag = "0.8.0" }
 stackable-trino-crd = { path = "../crd" }
+stackable-regorule-crd = { git = "https://github.com/stackabletech/regorule-operator.git", branch = "added_cluster_ip_service" }
```
Comment (maltesander) [inline rust/operator-binary/Cargo.toml:14]: That should be `main` branch for now until we tag it.

**Review by maltesander — CHANGES_REQUESTED** [review]
LGTM. Switching the branch should fix the errors.

File: rust/operator-binary/Cargo.toml (hunk 1 above)
Comment (teozkr) [reply rust/operator-binary/Cargo.toml:14]: Oh right, guess I still had the PR branch cached locally.

**Review by maltesander — APPROVED** [review]
LGTM.

**Comment by maltesander** [comment]
bors merge

**Pull request merged**